### ISSUE TITLE
Recipes registered in PreInt phase 1.7.10

### ISSUE THREAD

**Issue by 31trainman** [defeatedcrow/AppleMilkTea_1.7.X#1]
Some of your mods items/blocks are registered in the Preint phase leading to them not being able to be seen in nei via the r key/other problems http://pastebin.com/UHGcWZxH

Latest apple milk tea for 1.7.10
Mods List: http://pastebin.com/ss7RbZ2k

Forge 1614 mc 1.7.10
Java 8 update 91
Mac os x 10.11.4

**Comment by defeatedcrow** [comment]
You understand the meaning of what you write?
What You want?
We can not respond to this report. It did not caused by my mods.

First, 

> Latest apple milk tea for 1.7.10

Write the version specifically!
this is the repository of OLD VERSION OF AMT! NOT OF AMT2!!
Why you used OLD AMT repository??? I CANNOT identify your using version.

Second, 

> Some of your mods items/blocks are registered in the Preint phase

Items and blocks registration in preinit are recommended. PLEASE DO NOT SAY TO ME. PLEASE SAY TO THE FORGE TEAM THAT STOP PREINIT REGISTRATION. You are saying very strange things.

Finally,
The log you show is not include the error caused by my mod. Errors occurred by another mods. Why you sent this issue to me? Do you understand which mod caused the errors??

In conclusion, I do not deal with this problem.

**Comment by 31trainman** [comment]
Sorry was tired and didn't realize this want the amt2 github, il open a fresh issue on the right github

**Comment by defeatedcrow** [comment]
No needed.

Errors occurred by another mods.
I do not deal with this problem. That's all.
